source/user-manual/reference/ossec-conf/wodle-s3.rst

```diff
@@ -332,6 +333,145 @@ Name of AWS organization. Only works with CloudTrail buckets.
+| **Default value**  | All accounts.                             |
++--------------------+-------------------------------------------+
+| **Allowed values** | Comma list of 12 digit AWS Account ID     |
++--------------------+-------------------------------------------+
+
+
+Service\\aws_account_alias
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A user-friendly name for the AWS account.
+
++--------------------+-----------------------------+
+| **Default value**  | N/A                         |
++--------------------+-----------------------------+
+| **Allowed values** | Any string                  |
++--------------------+-----------------------------+
+
+Service\\access_key
+^^^^^^^^^^^^^^^^^^^
+
+The access key ID for the IAM user with the permission to access the service.
+
++--------------------+--------------------------+
+| **Default value**  | N/A                      |
++--------------------+--------------------------+
+| **Allowed values** | Any alphanumerical key.  |
++--------------------+--------------------------+
+
+Service\\aws_log_groups
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 4.0.0
+
+A comma-delimited list of log group names from where the logs should be extracted. Only works for CloudWatch Logs service.
+
++--------------------+------------------------------------------------+
+| **Default value**  | All regions                                    |
++--------------------+------------------------------------------------+
+| **Allowed values** | Comma-delimited list of valid log group names  |
++--------------------+------------------------------------------------+
+
+Service\\secret_key
+^^^^^^^^^^^^^^^^^^^
+
+The secret key created for the IAM user with the permission to access the service.
+
++--------------------+--------------------------+
+| **Default value**  | N/A                      |
++--------------------+--------------------------+
+| **Allowed values** | Any alphanumerical key.  |
++--------------------+--------------------------+
+
+Service\\aws_profile
+^^^^^^^^^^^^^^^^^^^^
+
+A valid profile name from a Shared Credential File or AWS Config File with the permission to access the service.
+
++--------------------+--------------------+
+| **Default value**  | N/A                |
++--------------------+--------------------+
+| **Allowed values** | Valid profile name |
++--------------------+--------------------+
+
+Service\\iam_role_arn
+^^^^^^^^^^^^^^^^^^^^^
+
+A valid role arn with permission to access the service.
+
++--------------------+----------------+
+| **Default value**  | N/A            |
++--------------------+----------------+
+| **Allowed values** | Valid role arn |
++--------------------+----------------+
+
+Service\\only_logs_after
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 4.0.0
+
+A valid date, in YYYY-MMM-DD format, that only logs from after that date will be parsed.  All logs from before that date will be skipped. Only works for CloudWatch Logs service.
```

Comment:
This format was chosen because is the one currently used for the rest of the supported buckets and services. We should not use a different format for each supported service. Changing all of them to use a new format will have negative impact on Wazuh users.